README.md

```diff
@@ -23,14 +24,26 @@ It's significant achievement when taking into account:
 * reduced Total Cost of Ownership with only one code base to maintain
 * naturally synchronized development process across all the teams
 (backend, presentation logic, frontend, mobile)
-* the same features release for all the platforms
+* the same features released for all the platforms
 * and the time-to-market for new value is shorter
 
-Just to name a few.
-Unlike in other cross-platform development methodologies, here UI developers are given full
-control over look and feel of their applications. I also believe that such an approach,
-abstracting application presentation logic from specific platform, brings much better overall
-architecture which pays off in the future.
+Just to name a few. Usually cross-platform development tools fall into these categories:
+
+* abstraction over native UI components and IO operations (specialized API
+accessible from any possible programming language which is later either interpreted
+or compiled for specific platform)
+* HTML+CSS forced as UI on mobile 
+
+But there is a third way, where only presentation logic code is shared and UI rendering
+and IO stays native. Where iOS, Web and Android developers can customize the app in every
+detail.
+
+This approach seems to be the most demanding one in terms of software architecture. Common business
+logic cannot operate on technical terms. Mouse click or touch event are becoming
```

Comment:
What does "operate on technical terms" mean? Do you mean that the business logic cannot be tightly coupled to UI events?